CMakeLists.txt

```diff
@@ -58,7 +58,14 @@ function(cctz_target_set_cxx_standard target)
   endif()
 endfunction()
 
-add_library(cctz
+set(CCTZ_HDRS
+  include/cctz/time_zone.h
+  include/cctz/civil_time_detail.h
+  include/cctz/zone_info_source.h
+  include/cctz/civil_time.h
+  )
+add_library(cctz "")
```

Comment:
cctz supports CMake 2.8.12 I believe, and `target_sources`, with such syntax, is a recent addition IIRC.

Also, what is the benefit for cctz of removing the sources from the `add_library` call?

Reply:
At first I wanted to use `target_source()` with `PUBLIC` for header...